images/create-env/create-env

```diff
@@ -69,9 +76,7 @@ set +u
 eval "$( conda shell.posix activate base )"
 set -u
 
-# Use --copy to cut links to package cache.
-# (Which is esp. important if --strip or --remove-files are used!)
-${conda_impl} create --yes --copy --prefix="${prefix}" "${@}"
+CONDA_YES=1 ${conda_impl} ${create_command} --prefix="${prefix}" "${@}"
```

Comment:
Ah, I just remembered we could also use
```suggestion
CONDA_ALWAYS_COPY=1 CONDA_YES=1 ${conda_impl} ${create_command} --prefix="${prefix}" "${@}"
```
and drop the `--copy` from the default command.
I'll change that later.